### PR TITLE
Task 68459 detail article, add comment

### PR DESCRIPTION
**Pull request by huuson9**
## Related Tickets
https://edu-redmine.sun-asterisk.vn/issues/68077
https://edu-redmine.sun-asterisk.vn/issues/68459

## WHAT (optional)
- Xem chi tiết từng bài viết 
- viết comment bên dưới

## HOW
- Tạo form comment
- Sửa file article_details
- sửa file view
- thay đổi dường dẫn url
## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
image
image

### CODE REVIEW

**Review by DragStark — APPROVED** [review]

**Comment by huuson9** [comment]
ready

**Comment by huuson9** [comment]
ready

File: CMS/home/views.py [hunk 1]
```diff
@@ -26,15 +30,28 @@ def updateUser(request, pk):
...
     article = Article.objects.all().order_by('-created_at')
-    context = {'article': article}
+    context = {'article': article, }
     return render(request, 'index.html', context)
 
 def articleDetail(request, pk): 
     article = get_object_or_404(Article, id=pk)
-    context = {'article': article}
+    comments = Comment.objects.filter(article=article)
+    # context = {'article': article, 'new_comment': new_comment, 'comment_form': comment_form}
```
Comment (huongnt-2545) [inline CMS/home/views.py:43]: xóa comment

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**